app/models/user.rb

```diff
@@ -565,6 +565,47 @@ def reactions_to
     Reaction.for_user(self)
   end
 
+  def latest_activity
+    return unless registered == true
+
+    latest_activity = last_comment_at || last_article_at || latest_article_updated_at ||
+      last_reacted_at || profile_updated_at || last_moderation_notification
+
+    if latest_activity == Time.zone.today
+      "Today #{latest_activity.strftime('%b, %Y')}"
+    elsif latest_activity == Date.yesterday
+      "Yesterday #{latest_activity.strftime('%b, %Y')}"
+    else
+      latest_activity.strftime("%d %b, %Y")
+    end
+  end
+
+  def current_status
```

Comment:
In the designs for the Member Index View, we only show the below roles in the "Status" column, however another suspended role does exist -- a user can also be "Comment suspended." Should we include "Comment suspended" as well, or leave the `current_status` method as-is?

cc @anujbhavsar96 @aitchiss @Ridhwana

Reply:
Those Base roles are status. It isn't obvious because they are part of roles. So in the member list view, we should display whatever status we have in the member detail view.

<img width="651" alt="Screenshot 2022-03-29 at 9 01 46 AM" src="https://user-images.githubusercontent.com/84626054/160527819-aab745ff-7add-4d96-922b-75a560a8fb36.png">

I should have covered all the cases in design, so it doesn't get confusing. Taking a note for future✨

Reply:
Thanks for the clarification, @anujbhavsar96! I'll add "Comment Suspended" to the method as well.